src/prefect/tasks/monte_carlo/monte_carlo_lineage.py

```diff
@@ -0,0 +1,469 @@
+import json
+from typing import Any, Dict, List
+from prefect.tasks.monte_carlo.client import MonteCarloClient
+
+import prefect
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class MonteCarloCreateOrUpdateLineage(Task):
+    """
+    Task for creating or updating a lineage node for the given source and destination nodes,
+    as well as for creating a lineage edge between those nodes.
+
+    Args:
+        - source (dict, optional): a source node configuration. Expected to include the following
+        keys: `node_name`, `object_id`, `object_type`, `resource_name`, and optionally also
+        `metadata_key` and `metadata_value`.
```

Comment:
Can you indent the extra lines? This goes for all docstrings.
```suggestion
        - source (dict, optional): a source node configuration. Expected to include the following
            keys: `node_name`, `object_id`, `object_type`, `resource_name`, and optionally also
            `metadata_key` and `metadata_value`.
```

Reply:
Thanks! Should be fixed in the latest commit